__tests__/product/product.query.test.js

```diff
@@ -4,8 +4,7 @@ const client = require('../../utils/apollo-test-client');
 const {
   badProductId,
   newProduct,
-  expectedResult,
-} = require('./product.variables');
+} = require('../../tests-variables/product.variables');
```

Comment:
There is no need to move `product` variables to another folder, please, just hold product variables inside `product` folder, in tests

Reply:
moved and deleted separate folder